src/ruby/puppetserver-lib/puppet/server/puppet_config.rb

```diff
@@ -15,9 +15,13 @@ def self.initialize_puppet(puppet_config)
     #
     # `config` is a map whose keys are the names of the settings that we wish
     # to override, and whose values are the desired values for the settings.
+    # Key values that are not strings are omitted, allowing for keys in the
```

Comment:
Maybe leave the word "key" out here, it make me think we're referring to the first part of the pair.